client/my-sites/promote-post-i2/components/campaign-item-details/index.tsx

```diff
@@ -88,6 +89,127 @@ const getExternalTabletIcon = ( fillColor = '#A7AAAD' ) => (
 	</span>
 );
 
+const trafficIcon = () => (
```

Comment:
In general, I think it would be better to have images in their own files and import them?

It might make them easier to find and make sense for reuse.  It also fills this file with a lot of non-readable code.

It doesn't need to block the shipping of this though, just an opinion 👍

Reply:
hm you have a point. but i am not sure if this also needs to come from the endpoint (the actual image should be in the dsp side I guess)

@sbarbosa what do you think?

Reply:
> I think it won't be a problem to apply the improvement that James mentioned here.

ok removed the icons from the index file and added them as individual components in this commit

https://github.com/Automattic/wp-calypso/pull/94757/commits/d605e5b52980751183cb7b9dcbe1a1b12bc06095